h2o-py/tests/testdir_javapredict/pyunit_PUBDEV_7185_GAM_mojo_gaussian.py

```diff
@@ -9,8 +9,8 @@ def gam_gaussian_mojo():
     h2o.remove_all()
     NTESTROWS = 200    # number of test dataset rows
     PROBLEM="gaussian"
-    params = set_params()   # set deeplearning model parameters
-    df = pyunit_utils.random_dataset(PROBLEM, missing_fraction=0.001)   # generate random dataset
+    params = set_params() 
+    df = pyunit_utils.random_dataset(PROBLEM, seed=1, missing_fraction=0.5) 
```

Comment:
Remove the seed.  I found that you can set the seed and it will make the test pass without fixing anything.
Add in the following change:

    exclude_list = {"response"}
    exclude_list.add(params["gam_columns"][0])
    x = list(set(df.names) - exclude_list)

replace the save in.csv with this line:
    h2o.download_csv(test, os.path.join(TMPDIR, 'in.csv'))  # save test file, h2o predict/mojo use same file

Reply:
done

Reply:
I kept the seed, random tests are not good, tests should be deterministic, even with the seed fails non-deterministically, need to investigate more